src/workloads/transactions/LLTInsertLowLtc.yml

```diff
@@ -4,6 +4,14 @@ Description: |
   Workload to Benchmark the effect of LongLivedTransactions on an Insert workload.
 
 GlobalDefaults:
+  # Start: The following block gets updated for the other configurations.
```

Comment:
Parameters were moved together to the top of the file. This is the source for all the other LLTInsert*.yml files.